lib/advert/ui/pages/admin_page/admin_page.dart

```diff
@@ -32,100 +32,96 @@ class AdvertAdminPage extends HookConsumerWidget {
     final selectedAnnouncers = ref.watch(announcerProvider);
     final selectedAnnouncersNotifier = ref.read(announcerProvider.notifier);
     return AdvertTemplate(
-      child: Refresher(
```

Comment:
There is no Refresher on the new version

Reply:
ColumnRefresher is now placed after the AsyncChild so that you can still encapsulate page elements while loading and also directly set children elements of the ColumnRefresher to optimize widget building

Reply:
Nice one then !